src/utils/EDA.py

```diff
@@ -0,0 +1,177 @@
+        sort (bool, optional): Whether to sort the values in data before creating the histogram. Defaults to True.
+        xlabel (str, optional): The label for the x-axis.
+        ylabel (str, optional): The label for the y-axis.
+        title (str, optional): The title of the histogram.
+
+    Returns:
+        None
+    """
+    if sort:
+        data = np.sort(data)
+
+    plt.hist(data, bins=bin, orientation=orientation)
+    plt.xlabel(xlabel)
+    plt.ylabel(ylabel)
+    plt.title(title)
+    plt.show()
+
+
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+
+def create_stacked_histogram(df, column1, column2, bins=10, sort=True, title="", xlabel="", ylabel="Frequency", orientation="vertical"):
+    """
+    Creates a stacked histogram of the given columns in the DataFrame.
+
+    Args:
+        df (pandas DataFrame): The DataFrame to create the histogram for.
+        column1 (str): The name of the first column to plot.
+        column2 (str): The name of the second column to plot.
+        bins (int, optional): The number of bins to use in the histogram.
+        sort (bool, optional): Whether to sort the data before creating the histogram. Defaults to True.
+        orientation (str, optional): The orientation of the histogram, either 'vertical' (default) or 'horizontal'.
+        xlabel (str, optional): The label for the x-axis.
+        ylabel (str, optional): The label for the y-axis.
+        title (str, optional): The title of the histogram.
+
+    Returns:
+        None
+    """
+    if sort:
+        df = df[[column1, column2]].apply(lambda x: np.sort(x), axis=0)
+    
+    # Create histogram of first column
+    plt.hist(df[column1], bins=bins, alpha=0.5, label=column1, orientation=orientation)
+
+    # Create histogram of second column
+    plt.hist(df[column2], bins=bins, alpha=0.5, label=column2, orientation=orientation)
+
+    # Add legend and labels
+    plt.legend(loc='upper right')
+    plt.xlabel(xlabel)
+    plt.ylabel(ylabel)
+    plt.title(title)
+    plt.show()
+
+
+
+
+def create_boxplot(df, col_name):
+    """
+    Create a boxplot for a given column of a pandas DataFrame.
+
+    Args:
+        df (pandas.DataFrame): The DataFrame to plot.
+        col_name (str): The name of the column to plot.
+
+    Returns:
+        None
+    """
+    fig, ax = plt.subplots()
+    ax.boxplot(df[col_name].dropna())
+
+    plt.text(x = 1.2, y=df[col_name].min(), s='min')
+    plt.text(x = 1.2, y=df[col_name].quantile(0.25), s='Q1')
+    plt.text(x = 1.2, y=df[col_name].median(), s='median(Q2)')
+    plt.text(x = 1.2, y=df[col_name].quantile(0.75), s='Q3')
+    plt.text(x = 1.2, y=df[col_name].max(), s='max')
+
+    #add the graph title and axes labels
+    ax.set_title(f'Boxplot of {col_name}')
+    ax.set_ylabel(col_name)
+    plt.show()
+
+
+
+def create_scatterplot(x, y, xlabel='', ylabel='', title=''):
```

Comment:
As previously, could you add the type hint for returned value and the arguments? Also, the title, x-label and y-label inputs are not optional arguments...